oozie-to-airflow/tests/converter/test_oozie_parser.py

```diff
@@ -238,6 +238,38 @@ def test_parse_action_node_pig_with_file_and_archive(self):
         self.assertEqual(["myNameNode/test_dir/test.txt#test_link.txt"], p_op.mapper.hdfs_files)
         self.assertEqual(["myNameNode/test_dir/test2.zip#test_zip_dir"], p_op.mapper.hdfs_archives)
 
+    def test_parse_mapreduce_node(self):
+        self.parser.params = {
+            "nameNode": "hdfs://",
+            "dataproc_cluster": "mycluster",
+            "gcp_region": "europe-west3",
+        }
+        node_name = "mr-node"
+        # language=XML
+        xml = """
+<action name='{node_name}'>
+    <map-reduce>
+        <name-node>hdfs://</name-node>
+        <prepare>
+            <delete path="hdfs:///user/mapreduce/examples/apps/mapreduce/output"/>
+        </prepare>
+    </map-reduce>
+    <ok to="end"/>
+    <error to="fail"/>
+</action>
+""".format(
```

Comment:
We agreed that we use f strings.

Reply:
Same story as https://github.com/GoogleCloudPlatform/oozie-to-airflow/pull/159#discussion_r285870131: it's copied from other existing tests. I'd suggest an edit in a separate PR.